pvlib/shading.py

```diff
@@ -342,3 +342,213 @@ def projected_solar_zenith_angle(solar_zenith, solar_azimuth,
+    surface_to_axis_offset : numeric, default 0
+        Distance between the rotating axis and the collector surface.
+    cross_axis_slope : numeric, default 0
+        Angle of the plane containing the rows' axes from
+        horizontal. Right-handed rotation with respect to ``axis_azimuth``.
+        In degrees :math:`^{\circ}`.
+    shading_tracker_rotation : numeric, optional
+        Right-handed rotation of the tracker casting the shadow, with respect
+        to ``axis_azimuth``. In degrees :math:`^{\circ}`.
+
+    Returns
+    -------
+    shaded_fraction : numeric
+        The fraction of the collector width shaded by an adjacent row. A
+        value of 1 is completely shaded and zero is no shade.
+
+    Notes
+    -----
+    All length parameters must have the same units to produce a reasonable
+    result.
+
+    Parameters are defined as follow:
+
+    .. figure:: ../../_images/Anderson_Jensen_2024_Fig3.png
+        :alt: Diagram showing the two trackers and the parameters of the model.
+
+        Figure 3 of [1]_. See correspondence between this nomenclature and the
+        function parameters in the table below.
+
+    +------------------+----------------------------+---------------------+
+    | Symbol           | Parameter                  | Units               |
+    +==================+============================+=====================+
+    | :math:`\theta_1` |``shading_tracker_rotation``|                     |
+    +------------------+----------------------------+                     |
+    | :math:`\theta_2` | ``shaded_tracker_rotation``| Degrees             |
+    +------------------+----------------------------+ :math:`^{\circ}`    |
+    | :math:`\beta_c`  | ``cross_axis_slope``       |                     |
+    +------------------+----------------------------+---------------------+
+    | :math:`p`        | ``pitch``                  | Any consistent      |
+    +------------------+----------------------------+ length unit across  |
+    | :math:`\ell`     | ``collector_width``        | all these           |
+    +------------------+----------------------------+ parameters, e.g.    |
+    | :math:`z_0`      | ``surface_to_axis_offset`` | :math:`m`.          |
+    +------------------+----------------------------+---------------------+
+    | :math:`f_s`      | Return value               | Dimensionless       |
+    +------------------+----------------------------+---------------------+
+
+    Examples
+    --------
+    Fixed-tilt south-facing array on flat terrain
+    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    Tilted tracker with a pitch of :math:`1.5m`, a collector width of
+    :math:`2m`, and tracker rotations of :math:`30^{\circ}`. In the morning.
+
+    >>> shaded_fraction1d(solar_zenith=80, solar_azimuth=104.5,
+    ...     axis_azimuth=90, shaded_tracker_rotation=30,
+    ...     shading_tracker_rotation=30, collector_width=2, pitch=1,
```

Comment:
I think most (all?) realistic scenarios will have `pitch > collector_width`.  For fixed-tilt arrays it is technically possible to have `pitch < collector_width`, but I am not sure anyone builds such a system.  For trackers it is not possible, as the modules will collide with each other at `rotation=0` :)

Let's increase these `pitch` values to 3 or 4 meters.  

(note also that the text above the code says it is 1.5 m, not 1)

Reply:
Yup, I didn't think much about it.